src/consensus/pbft/pbft.h

```diff
@@ -148,7 +144,7 @@ namespace pbft
 
       // Adding myself
       PrincipalInfo my_info;
-      my_info.id = local_id;
```

Comment:
Not very fond of replacing local_id with self, local_id was clearly the id of the local node, whereas self could be just about anything.

Reply:
Thanks I'll change it back. I was undecided on whether to change it or not :)